mirobo/device.py

```diff
@@ -25,14 +25,20 @@ def __repr__(self):
                                                  self.data["token"])
 
     @property
-    def netif(self):
+    def netif(self) -> str:
```

Comment:
IIRC these are not strings but rather dicts, maybe it's a good idea to add an example output into the class' docstring.

Reply:
My fault. I wil add a proper docstring for the future.

Reply:
Not a problem. Looks like I lost my connectivity to my raspberry pi running as a controller, so I can't do much testing on this for now. If you can, could you please add the output of the info() to the doctstring of DeviceInfo class (that will also help when creating unit tests at some point).